src/pages/rental/rental.ts

```diff
@@ -62,18 +62,63 @@ export class RentalPage {
    * Pushes page on nav to allow users to finish renting the items.
    */
   onContinue() {
-    this.navCtrl.push(RentalDetailsPage);
+    let items;
+    this.items.take(1).subscribe(i => items = i.rentals);
```

Comment:
What does `i` mean here? I would use a more descriptive variable name unless it means "index" or something equally non-meaningful.